addons/l10n_es_pos/static/src/overrides/models/pos_store.js

```diff
@@ -6,7 +6,7 @@ patch(PosStore.prototype, {
     getReceiptHeaderData(order) {
         const result = super.getReceiptHeaderData(...arguments);
         result.is_spanish = this.config.is_spanish;
-        result.simplified_partner_id = this.config.simplified_partner_id[0];
+        result.simplified_partner_id = this.config.simplified_partner_id.id;
```

Comment:
do we make sure that this `simplified_partner_id` record is loaded?

Reply:
I added this: 

```
    async afterProcessServerData() {
        await super.afterProcessServerData(...arguments);

        // We need to update the partner list after the pos config is loaded
        if (this.config.is_spanish && !this.config.simplified_partner_id) {
            await this._loadPartners([this.config.raw.simplified_partner_id]);
        }
    },
```